lib/internal/bootstrap/switches/is_main_thread.js

```diff
@@ -158,6 +158,7 @@ function getStdin() {
     case 'FILE':
       const fs = require('fs');
       stdin = new fs.ReadStream(null, { fd: fd, autoClose: false });
+      stdin.pause();
```

Comment:
Where does this get resumed?

Reply:
Nowhere unless the user attaches a `data` handler - that is the point
This stream is created behind the scenes in the `stdin` getter - so the point is that it should not have any side effects unless you actually use it

Reply:
pause sets flowing to false...

Reply:
attaching a data handler is not supposed to unpause a paused stream.

Reply:
Readable is tricky :(

Reply:
`Readable` is resumed upon attaching a `data` listener:
<https://github.com/nodejs/node/blob/348b7a9606da61fbb5437e27f479363114ecf4eb/lib/internal/streams/readable.js#L863>